config/backend-config/backend-config_test.go

```diff
@@ -3,7 +3,8 @@ package backendconfig
 import (
 	"bytes"
 	"encoding/json"
-	"errors"
+
```

Comment:
Why do we need to add errors package in every file?

Reply:
It's not added. It's replaced.